deployments/grafana-demo/02-auto-instrument.yml

```diff
@@ -0,0 +1,162 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-agent-config
+data:
+  agent-config.river: |
+    otelcol.receiver.otlp "default" {
+      grpc {}
+      http {}
+    
+      output {
+        metrics = [otelcol.processor.batch.default.input]
+        traces = [otelcol.processor.batch.default.input]
+      }
+    }
+      
+    otelcol.processor.batch "default" {
+      output {
+        metrics = [otelcol.exporter.prometheus.default.input]
+        traces  = [otelcol.exporter.otlp.tempo.input]
+      }
+    }
+      
+    otelcol.exporter.prometheus "default" {
+      forward_to = [prometheus.remote_write.mimir.receiver]
+    }
+      
+    prometheus.remote_write "mimir" {
+      endpoint {
+        url = "https://" + env("MIMIR_USER") + ":" + env("GRAFANA_API_KEY") + "@" + env("MIMIR_ENDPOINT") + "/api/prom/push"
+      }
+    }
+      
+    otelcol.exporter.otlp "tempo" {
+      // Send traces to a locally running Tempo without TLS enabled.
+      client {
+        endpoint = env("TEMPO_ENDPOINT")
+        auth     = otelcol.auth.basic.creds.handler
+      }
+    }
```

Comment:
Can we use local graphana or prometheus ? In opentelemetry-demo, we use same https://github.com/open-telemetry/opentelemetry-demo/blob/5f7517aad8e2f1365c7d4b173194bfdba2da4489/docker-compose.yml#L624